### PR TITLE
Update docs/run.md

### LINKED ISSUE #623: Documentation for `clasp run` is outdated
> Please search [existing issues](https://github.com/google/clasp/issues) before creating a new one.
>
> DON'T DELETE THIS! FILL THESE SECTIONS OUT!

## Expected Behavior

The documentation should show how to get the project id in a new script.

## Actual Behavior

In [run.md](https://github.com/google/clasp/blob/master/docs/run.md) , how to get Project ID is presented as follows

> Add a projectId to your .clasp.json. You can find your Project ID via:
> - clasp open
> - In the menu, click Resources > Cloud Platform project...
> - Copy the project ID. Example: project-id-7006438181792679938.
> - Paste this in projectId in your .clasp.json.

However we can't find Project ID in `Cloud Platform project`.

image

In English:
> This script has a Cloud Platform project managed by Google Script.

See also : https://developers.google.com/apps-script/guides/cloud-platform-projects

## Steps to Reproduce the Problem

1.
1.

## Specifications

- Node version (`node -v`):
- Version (`clasp -v`):
- OS (Mac/Linux/Windows):

### PR DESCRIPTION
**Pull request by takanakahiko**
Fixes #623 

Among the multiple methods, I wrote the simplest one.

- [ ] `npm run test` succeeds.
- [ ] `npm run lint` succeeds.
- [x] Appropriate changes to README are included in PR.

### CODE REVIEW

File: docs/run.md [hunk 1]
```diff
@@ -21,10 +21,15 @@ To use `clasp run`, you need to complete 4 steps:
 1. Ensure you have upgraded to the latest version
     - `clasp -v`
 1. Add a `projectId` to your `.clasp.json`. You can find your Project ID via:
+    - [Create GCP project and get ProjectId](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+    - get `Project ID` and `Project number`. Example: `my-sample-project-191923.` and `314053285323`.
+    - Paste `Project ID` in `projectId` in your `.clasp.json`.
+1. Apply a `projectId` to your GAS project
+    - Open `https://console.developers.google.com/apis/credentials/consent?project=[projectId]`
+    - Enter `Application name` (example: `clasp run oauth`) and click `save`
```
Comment (grant) [inline docs/run.md]: - Set `Application name` to `clasp project` and click `save`.

File: docs/run.md [hunk 2]
```diff
@@ -21,10 +21,15 @@ To use `clasp run`, you need to complete 4 steps:
 1. Ensure you have upgraded to the latest version
     - `clasp -v`
 1. Add a `projectId` to your `.clasp.json`. You can find your Project ID via:
+    - [Create GCP project and get ProjectId](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
```
Comment (grant) [inline docs/run.md]:
```
[Create a GCP project](...)
```

File: docs/run.md [hunk 3]
```diff
@@ -21,10 +21,15 @@ To use `clasp run`, you need to complete 4 steps:
...
+    - get `Project ID` and `Project number`. Example: `my-sample-project-191923.` and `314053285323`.
```
Comment (grant) [inline docs/run.md]: - Record the `Project ID` and `Project number`.
  - (Example: ...)

File: docs/run.md [hunk 4]
```diff
@@ -21,10 +21,15 @@ To use `clasp run`, you need to complete 4 steps:
...
+    - Paste `Project ID` in `projectId` in your `.clasp.json`.
```
Comment (grant) [inline docs/run.md]: Run the command with your project ID: `clasp setting projectId <PROJECT_ID>`

File: docs/run.md [hunk 5]
```diff
@@ -21,10 +21,15 @@ To use `clasp run`, you need to complete 4 steps:
...
+1. Apply a `projectId` to your GAS project
```
Comment (grant) [inline docs/run.md]: 1. Set the `projectId` to your Apps Script project

**Comment by takanakahiko** [comment]
Thanks!
This is a very helpful review, because I'm not good at English.

**Comment by takanakahiko** [comment]
@grant  Please review.

**Comment by grant** [comment]
Looks pretty good. Thanks!

**Comment by mikearche** [comment]
Should the projectId section be updated as well?
I created a project using clasp, and when I went to figure out how to get logging working I was pretty confused.

See: https://github.com/google/clasp#projectid-optional

**Comment by takanakahiko** [comment]
@mikearche Thanks for your report. I' create PR.

**Pull request merged**